app/templates/electron/ts/main-process/electron-main.ts

```diff
@@ -0,0 +1,73 @@
+import { app, BrowserWindow, nativeTheme } from 'electron'
+import { unlinkSync } from 'fs'
+import { join } from 'path'
+
+try {
+  if (
+    process.platform === 'win32' &&
+    nativeTheme.shouldUseDarkColors === true
+  ) {
+    unlinkSync(join(app.getPath('userData'), 'DevTools Extensions'))
+  }
+} catch (_) {}
+
+/**
+ * Set `__statics` path to static files in production;
+ * The reason we are setting it here is that the path needs to be evaluated at runtime
+ */
+
+if (process.env.PROD) {
+  global.__statics = __dirname
+}
+
+let mainWindow: null | BrowserWindow = null
```

Comment:
Usually `null` is placed after the real type. In TS is also better to avoid `null` altogether, using `undefined` instead if you're not forced to do so.

So just adding `let mainWindow: BrowserWindow | undefined` should be fine

Reply:
I used `null1 because of the code used in the end:
```ts
mainWindow === null
```